### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by borkdude**

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
 
-How to hack:
+## Issue
+
+Before writing any code, please create an issue first that describes the problem
+you are trying to solve with alternatives that you have considered. A little bit
+of prior communication can save a lot of time on coding. Keep the problem as
+small as possible. If there are two problems, make two issues. We discuss the
+issue and if we reach an agreement on the approach, it's time to move on to a
+PR.
```
Comment (PEZ) [inline CONTRIBUTE.md:10]: This is great. Maybe even add something about that it sometimes is best to start with a discussion thread, to collaboratively figure out what the problem(s) is/are?

Comment (borkdude) [reply CONTRIBUTE.md:10]: I'll add it

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Please answer the following questions and leave the below in as part of your PR.
+
+- [ ] I have read the [developer documentation](https://github.com/BetterThanTomorrow/joyride/blob/master/CONTRIBUTE.md).
+
+- [ ] This PR corresponds to an [issue with a clear problem statement](https://github.com/babashka/babashka/blob/master/doc/dev.md#start-with-an-issue-before-writing-code).
+
+- [ ] This PR contains a [test](https://github.com/BetterThanTomorrow/joyride/tree/master/vscode-test-runner/workspace-1/.joyride/src/integration_test) to prevent against future regressions
+
+- [ ] I have updated the [CHANGELOG.md](https://github.com/BetterThanTomorrow/joyride/blob/master/CHANGELOG.md) file with a description of the addressed issue.
```
Comment (PEZ) [inline .github/pull_request_template.md:9]: To make more of these fit our publish script from start:

```suggestion
- [ ] I have updated the **\[Unreleased\]** section of the [CHANGELOG.md](https://github.com/BetterThanTomorrow/joyride/blob/master/CHANGELOG.md) file with a link to the addressed issue.
```

File: CONTRIBUTE.md [hunk 3]
```diff
@@ -1,6 +1,34 @@
...
+
+## PR
+
+Follow up with a pull request Post a corresponding PR with the smallest change
+possible to address the issue. Then we discuss the PR, make changes as needed
+and if we reach an agreement, the PR will be merged.
+
+Please do not use `git push --force` on your PR branch for the following reasons:
```
Comment (PEZ) [inline CONTRIBUTE.md:20]: I find that sometimes I've done something that only a force push can fix. Not saying we should not ask for this. More noticing this, and maybe it is about some gap in my Git-fu.

**Review by PEZ — COMMENTED** [review]
LGTM! Funny, I sat down at my computer with the plan to write a small thing about how to hack on Joyride. 😄

**Pull request merged**